### PR TITLE
Add deliveries of leaderless BFT consensus protocol

### PR DESCRIPTION
**Pull request by gitferry**
This PR added all the deliveries of the application [leaderless-bft-consensus](https://github.com/vechain/grant-program/blob/master/applications/leaderless_bft_consensus.md).

### CODE REVIEW

**Review by zzGHzz — APPROVED** [review]

File: accepted_grant_applications.md [hunk 1]
```diff
@@ -49,10 +49,10 @@ This page gives an overview of accepted grants and a link to their GitHubs. Keep
 
 # 2021
 
-| Project | Category | Link | Funds | First Delivery | Completed |  Milestone Status |
-| --- | :-: | :-: | :-: | :-: | :-: | :-: |
+| Project | Category | Link | Funds | First Delivery | Completed |          Milestone Status          |
+| --- | :-: | :-: | :-: | :-: | :-: |:----------------------------------:|
```
Comment (libotony) [inline accepted_grant_applications.md:53]:
```suggestion
| Project | Category | Link | Funds | First Delivery | Completed | Milestone Status |
| --- | :-: | :-: | :-: | :-: | :-: |:-:|
```

File: accepted_grant_applications.md [hunk 2]
```diff
@@ -49,10 +49,10 @@ This page gives an overview of accepted grants and a link to their GitHubs. Keep
...
-| [Leaderless Byzantine Fault Tolerant Consensus](https://github.com/vechain/grant-program/blob/master/applications/leaderless_bft_consensus.md) | Research |  [Applicantion](https://github.com/vechain/grant-program/blob/master/applications/leaderless_bft_consensus.md) | $15,200 |  |  | ![](https://geps.dev/progress/0) |
-| [Digital Village](https://github.com/vechain/grant-program/blob/master/applications/digital-village.md) | NFT Marketplace |  [Website](https://digitalvillage.io/) | $30,000 |   |   | ![](https://geps.dev/progress/0) |
-| [Village Protocol](https://github.com/vechain/grant-program/blob/master/applications/village-protocol.md) | Social Media |  [Website](https://digitalvillage.io/) | $30,000 |  |  | ![](https://geps.dev/progress/0) |
+| [An institutional map of VeChain governance](https://github.com/vechain/grant-program/blob/master/applications/map-of-vechain-governance.md) | Research  | [Applicantion](https://github.com/vechain/grant-program/blob/master/applications/map-of-vechain-governance.md)  | $29,646 | ✅ |   | ![](https://geps.dev/progress/33)  |
```
Comment (libotony) [inline accepted_grant_applications.md:55]:
```suggestion
| [An institutional map of VeChain governance](https://github.com/vechain/grant-program/blob/master/applications/map-of-vechain-governance.md) | Research  | [Applicantion](https://github.com/vechain/grant-program/blob/master/applications/map-of-vechain-governance.md)  | $29,646 | ✅ |   | ![](https://geps.dev/progress/33) |
```

File: accepted_grant_applications.md [hunk 3]
```diff
@@ -49,10 +49,10 @@ This page gives an overview of accepted grants and a link to their GitHubs. Keep
...
+| [Leaderless Byzantine Fault Tolerant Consensus](https://github.com/vechain/grant-program/blob/master/applications/leaderless_bft_consensus.md) | Research |  [Applicantion](https://github.com/vechain/grant-program/blob/master/applications/leaderless_bft_consensus.md) | $15,200 | ✅ | ✅ | ![](https://geps.dev/progress/100) |
+| [Digital Village](https://github.com/vechain/grant-program/blob/master/applications/digital-village.md) | NFT Marketplace |  [Website](https://digitalvillage.io/) | $30,000 |   |   |  ![](https://geps.dev/progress/0)  |
```
Comment (libotony) [inline accepted_grant_applications.md:57]:
```suggestion
| [Digital Village](https://github.com/vechain/grant-program/blob/master/applications/digital-village.md) | NFT Marketplace |  [Website](https://digitalvillage.io/) | $30,000 |   |   | ![](https://geps.dev/progress/0) |
```

File: accepted_grant_applications.md [hunk 4]
```diff
@@ -49,10 +49,10 @@ This page gives an overview of accepted grants and a link to their GitHubs. Keep
...
+| [Village Protocol](https://github.com/vechain/grant-program/blob/master/applications/village-protocol.md) | Social Media |  [Website](https://digitalvillage.io/) | $30,000 |  |  |  ![](https://geps.dev/progress/0)  |
```
Comment (libotony) [inline accepted_grant_applications.md:58]:
```suggestion
| [Village Protocol](https://github.com/vechain/grant-program/blob/master/applications/village-protocol.md) | Social Media |  [Website](https://digitalvillage.io/) | $30,000 |  |  | ![](https://geps.dev/progress/0) |
```

**Review by libotony — CHANGES_REQUESTED** [review]

**Comment by gitferry** [comment]
Hey @libotony thanks for your comments. In the last commit ([505b0c4](https://github.com/vechain/grant-program/pull/230/commits/505b0c4a7ee505a35a9add50d08b48cc155ab7e7)) I have changed the Ethereum address that we will use to receive USDT. Is there any change needed for this PR?

**Comment by libotony** [comment]
> Hey @libotony thanks for your comments. In the last commit ([505b0c4](https://github.com/vechain/grant-program/pull/230/commits/505b0c4a7ee505a35a9add50d08b48cc155ab7e7)) I have changed the Ethereum address that we will use to receive USDT. Is there any change needed for this PR?

Looks good, thanks!

**Review by libotony — APPROVED** [review]

File: milestone-delivery/deliveries/leaderless_bft_consensus_milestone_1.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Milestone Delivery :mailbox:
+
+
+* **PR Link:** https://github.com/vechain/grant-program/pull/2
+* **Milestone Number:** 2
```
Comment (libotony) [inline milestone-delivery/deliveries/leaderless_bft_consensus_milestone_1.md:5]:
```suggestion
* **Milestone Number:** 1
```

**Review by libotony — APPROVED** [review]

**Comment by NecoSherry** [comment]
Thank you very much for the delivery! We will proceed with the payment for your application promptly. We appreciate your patience in advance.

**Pull request merged**